sdk/schemaregistry/azure-data-schemaregistry/README.md

```diff
@@ -119,34 +120,37 @@ String schemaContent = "{\n"
     + "    ]\n"
     + "}";
 SchemaProperties schemaProperties = schemaRegistryClient.registerSchema("{schema-group}", "{schema-name}",
-    schemaContent, SerializationFormat.AVRO);
-System.out.println("Registered schema: " + schemaProperties.getSchemaId());
+    schemaContent, SchemaFormat.AVRO);
 ```
 
 ### Retrieve a schema ID
 Retrieve a previously registered schema ID from the Azure Schema Registry.
 
 <!-- embedme ./src/samples/java/com/azure//data/schemaregistry/ReadmeSamples.java#L77-L85 -->
 ```java
-TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
 
-SchemaRegistryClient schemaRegistryClient = new SchemaRegistryClientBuilder()
-    .endpoint("{schema-registry-endpoint")
-    .credential(tokenCredential)
-    .buildClient();
 
-SchemaProperties schemaProperties = schemaRegistryClient.getSchema("{schema-id}");
-System.out.println("Retrieved schema: " + schemaProperties.getSchemaName());
+
+
+
+
+
+
+
 ```
 ### Retrieve a schema
 Retrieve a previously registered schema's content from the Azure Schema Registry.
 
 <!-- embedme ./src/samples/java/com/azure//data/schemaregistry/ReadmeSamples.java#L92-L114 -->
 ```java
+
+ample for getting the schema id of a registered schema.
+
+ic void getSchemaId() {
```

Comment:
Looks like the code snippet did not get embedded correctly.

Reply:
Blegh! Thanks for that!